replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+suppressPackageStartupMessages(require(psych))
+suppressPackageStartupMessages(require(car))
+suppressPackageStartupMessages(require(MASS))
+suppressPackageStartupMessages(require(stargazer))
+
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
+Disliked_Music <- c("8(23%)","27(77%)", "")
+Condition <- c("AdvertisedPen", "NonAdvertisedPen", "Chi-Square")
+Freq_Table = cbind(Condition, Liked_Music, Disliked_Music)
+stargazer(Freq_Table, type = "text", title = "Study 3: Frequencies of Pen Choice for Each Conditon & Chi-Square")
+
+```
+
+(2) Explanation: 
+The table above is a reproduction of the frequencies and percentages of subjects' choice for advertised and non-advertised pen in the Liked Music & Disliked Music conditions, respectively, in Study 3. Note that at the bottom of the table is also the chi-square statistic, p-value and effect size (phi) for the data, indicating a medium-sized effect of music condition on pen choice. The table also demonstrates that the frequency of choice in the Disliked Music condition was in the hypothesized direction: Subjects tended to choose the non-advertised pen when made to listen to disliked music in an advertisement. Please note that the data represented above follows the exclusion criteria set forth in the published article. Specifically, any subjects who had participated in a prior version of the study were excluded (two subjects). In addition, 19 subjects were excluded for their so-called "deviant music evaluations". These subjects were foundd to either somewhat dislike the "liked music" (music attiude score below 3) or somewhat like the "disliked music" (music attitude score above 3).
+
+(3) Code for analyses & table:
+
+```{r eval= FALSE}
+
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+#head(dat)
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
+#dat_e
+#View(dat_e)
+
+## Analyses on ENTIRE sample:
+
+# Test whether music condition is independent of pen choice:
+# (1 == liked music, 2 == disliked music)
+# (0 == not chosen, 1 == chosen)
+tbl <- table(dat_e$music, dat_e$chose_advertised_pen) 
+tbl    
+
+# Chi-Square, no correction, no simulation:
+cs1 <- chisq.test(tbl, correct = FALSE)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .39, N = 91, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
```

Comment:
is this how the SUBSETTED data is different than the ENTIRE data? how is "too positive" and "too negative" operationalized?